tools/auto-fuzz/manager.py

```diff
@@ -503,15 +571,23 @@ def autofuzz_project_from_github(github_url,
                          oss_fuzz_base_project.project_name)):
         return False
 
-    # If this is a jvm target download maven once so we don't have to do it
-    # for each proejct.
+    # If this is a jvm target download maven and gradle once so we don't
+    # have to do it for each proejct.
     if language == "jvm":
+        # Download Maven
         MAVEN_URL = "https://downloads.apache.org/maven/maven-3/3.6.3/binaries/apache-maven-3.6.3-bin.zip"
```

Comment:
Also these types of constants, please keep them in the `constants.py` module or elsewhere e.g. at the top of the module